lib/components/molecules/Search/Search.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+/**
+ *
+ * Search
+ *
+ */
+import React, { useEffect, useState, useCallback } from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import Input from '../../atoms/Input';
+import Button from '../../atoms/Button';
+import List from '../../molecules/List';
+
+import styles from './Search.style';
+import type { Props } from './types';
+
+const Search = ({ className, inputProps, dataLists, children, renderListItem }: Props): Node => {
+  const [inputValue, setInputValue] = React.useState(inputProps.value || '');
+  const [searchLists, setSearchList] = React.useState([]);
+
+  const onInputChange = e => {
+    setInputValue(e.target.value);
+  };
+
+  const onSearchClick = () => {
+    if (inputValue.trim()) {
+      const lists = dataLists.filter(val => {
+        return val.toLowerCase().indexOf(inputValue.toLowerCase()) !== -1;
+      });
+      setSearchList(lists);
+    }
+    return false;
+  };
+
+  const keyPress = e => {
```

Comment:
You can do key press on list items too and on search field too, which one it is?

Reply:
It's in search field.